website/library/reference/how-to-review-someone's-work.md

```diff
@@ -0,0 +1,52 @@
+# How to be a great peer reviewer
+
+When you first start your technical writing journey, one of the many important skills you'll need to learn is how to do is peer reviews. It's a skill that will not only teach you how to collaborate with other technical writers and other people who interact with documentation, but it will also help you to become a better technical writer in the long run.
+
+## Tips for doing effective Peer Reviews
+
+When it comes to peer reviews, it can get confusing and overwhelming at times because you want to be effective, but at the same time you don't want to give too much feedback to point that the TA is confused. Consider using this checklist of things to keep in mind when doing peer reviews:
+
+### Content Review
+
+* **Overall structure and flow**:
+  * Are the steps in the correct order?
+  * Will the user be able to understand the steps?
+  * Have all the terms been defined at the first use?
+* **Accuracy and completeness**:
+  * Are all the steps accurate?
+  * Are there any steps missing or unclear?
+  * If you run through the document from start to finish, does it work?
```

Comment:
I think these points could do with some clarity. The later sections all seem quite self-evident, but I don't entirely know what "are there any steps... unclear" means. A lot of these points deal with the perspective of the end-user, and I think this is the hardest part for anyone to grok when they're deeply embroiled in the technical understanding and low-level functioning of a system. The author gets so far in the weeds they find it prohibitively difficult to see the other side.

Is there a way to rephrase these things to make them more concrete? I think everyone can agree with the idea that "content should aim to be understandable by the end user", but what that *means* is the real challenge.